contracts/modules/licensing/LicensingModule.sol

```diff
@@ -366,6 +374,24 @@ contract LicensingModule is
         uint256 licenseTermsId,
         Licensing.LicensingConfig memory licensingConfig
     ) external verifyPermission(ipId) whenNotPaused {
+        if (licenseTemplate == address(0) && licensingConfig.commercialRevShare != 0) {
+            revert Errors.LicensingModule__LicenseTemplateCannotZeroAddressForOverrideRoyaltyPercent();
```

Comment:
Can rename `LicensingModule__LicenseTemplateCannotZeroAddressForOverrideRoyaltyPercent` to `LicensingModule__LicenseTemplateCannotBeZeroAddressToOverrideRoyaltyPercent` ?

Reply:
rename in commit: [9ec413c](https://github.com/storyprotocol/protocol-core-v1/pull/269/commits/9ec413c9be8cf0be38b11fdaa5f37d4c580b5071)